sdk/openai/Azure.AI.OpenAI/tests/AssistantTests.cs

```diff
@@ -598,7 +598,7 @@ await foreach (ThreadMessage message in messages)
                 hasCake |= content.Text?.ToLowerInvariant().Contains("cake") == true;
                 foreach (TextAnnotation annotation in content.TextAnnotations)
                 {
-                    Console.WriteLine($"  --> From file: {annotation.InputFileId}, quote: {annotation.InputQuote}, replacement: {annotation.TextToReplace}");
```

Comment:
Out of curiosity, Was the removal of `TextAnnotation.InputQuote` intentional?

Reply:
It was; breaking change via the OpenAI spec: https://github.com/openai/openai-dotnet/blob/main/CHANGELOG.md#features-added